avocado/plugins/tap.py

```diff
@@ -156,5 +159,10 @@ def configure(self, parser):
                                        " directory. File will be named "
                                        "\"results.tap\".")
 
+        cmd_parser.output.add_argument('--tap-include-logs',
+                                       action='store_true', help='Include '
+                                       'test logs as comments in TAP output ('
+                                       '(%(default)s)')
```

Comment:
There's one extra opening parenthesis here.  Also, the predominant style when listing defaults is `'Current: %(default)s'`